source/assets/stylesheets/scss/_homepage.scss

```diff
@@ -386,4 +253,212 @@
   .go {
     text-transform: none;
   }
-}
\ No newline at end of file
+}
+
+
+.two-row-col {
```

Comment:
This CSS does not look like it's specific to the home page. Move this to `scss/general` maybe?